src/main/java/com/redhat/rhjmc/containerjfr/commands/internal/ExitCommand.java

```diff
@@ -69,6 +69,13 @@ public void validate(String[] args) throws FailedValidationException {
             cw.println(errorMessage);
             throw new FailedValidationException(errorMessage);
         }
+        for (String arg : args) {
+            if (arg == null) {
+                String errorMessage = "One or more arguments were null";
+                cw.println(errorMessage);
+                throw new FailedValidationException(errorMessage);
+            }
+        }
```

Comment:
For commands which expect 0 arguments, does it still make sense to make assertions about the contents of an empty array? They (should have) already validated that the array has length 0 at this point.

Reply:
My thinking was that since the code would be dead anyway in those cases, we could just put the check in every `validate()` method, to make it uniform and to guard against possible additions of parameters. I guess the tests would need to be added anyway in that case though, and unnecessary code is probably bad, so I'll get rid of it.

Reply:
That's a fair enough idea, although in that case it would make sense to do something like put the null validation into a superclass and then ensure that all validations call through it. ex. (rough pseudojava):

```
abstract class SuperClass {
  void doValidation(String[] args) {
    doNullValidation(args);
    validate(args);
  }
  void validate(String[] args);
}

class SubClass {
  void validate(String[] args) {
    doNumArgsValidation();
    doOtherValidations();
  }
}
```

Reply:
I ended up just getting rid of it since I think that's a little cleaner and since there won't be many commands eventually, so it's probably fine to just keep the validation simple.